### PR TITLE
Fixing home

### PR DESCRIPTION
**Pull request by azurast**

### CODE REVIEW

File: application/views/pages/homePage_view.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8">
+        <?php echo $css ?>
+        <title>Infinistyle</title>
+    </head>
+    <body>
+        <?php echo $header ?>
+        <?php echo $content ?>
```
Comment (cindmichelle) [inline application/views/pages/homePage_view.php]: contentnya dapet drmn ya nanti azura??

**Review by cindmichelle — APPROVED** [review]

**Review by cindmichelle — APPROVED** [review]
LGTM, ship it!

**Review by cindmichelle — APPROVED** [review]

File: application/views/includes/user/profile.php [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+                        </div>
+                        <div class="col-lg-4 order-lg-3 text-lg-right align-self-lg-center">
+                        </div>
+                        <div class="col-lg-4 order-lg-1">
+                        </div>
+                    </div>
+                    <div class="text-center" style="margin-top:16%;">
+                        <h3>Jessica Jones</h3>
+                        <div class="h6 font-weight-300"><i class="ni location_pin mr-2"></i>Gading Serpong</div>
+                        <div class="h6 mt-4"><i class="ni business_briefcase-24 mr-2"></i>Customer</div>
```
Comment (cindmichelle) [inline application/views/includes/user/profile.php]: test

File: application/views/includes/user/navbar.php [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+                <div class="row">
+                    <div class="col-6 collapse-brand">
+                        <a href="../../pages/dashboards/dashboard.html">
+                            <img src="../../assets/img/brand/blue.png">
+                        </a>
+                    </div>
+                    <div class="col-6 collapse-close">
+                        <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse" aria-controls="navbar-collapse" aria-expanded="false" aria-label="Toggle navigation">
+                            <span></span>
+                            <span></span>
```
Comment (cindmichelle) [inline application/views/includes/user/navbar.php]: i think we can remove this

**Review by cindmichelle — APPROVED** [review]
ok

File: application/views/includes/user/navbar.php (hunk 3 above)
Comment (azurast) [reply application/views/includes/user/navbar.php]: ok

File: application/controllers/shop/Details.php [hunk 4]
```diff
@@ -9,9 +9,13 @@ public function __construct(){
     }
 
     public function index(){
+
+        $id = $this->input->get('id');
+        $where = array('productID'=>$id);
```
Comment (azurast) [inline application/controllers/shop/Details.php]: mungkin salah disini si $where nya, tapi belum yakin cara benerinnya gimana

**Review by cindmichelle — APPROVED** [review]
test uda bs di merge belom? hehe

**Comment by azurast** [comment]
> 
> 
> test uda bs di merge belom? hehe

belom :(

**Review by cindmichelle — APPROVED** [review]

**Review by cindmichelle — APPROVED** [review]

**Pull request merged**